src/image/ImageAnnotator.ts

```diff
@@ -79,6 +80,7 @@ export default class ImageAnnotator extends BaseAnnotator {
                 height: `${referenceEl.offsetHeight}px`,
                 left: `${referenceEl.offsetLeft}px`,
                 top: `${referenceEl.offsetTop}px`,
+                transform: `rotate(${rotation}deg)`,
```

Comment:
Would this cause the `RegionCreator` to get rotated as well, including the `PopupCursor`?

Reply:
That’s right. We’re disabling creation for rotated images for now, though. See https://github.com/box/box-content-preview/pull/1232.

Reply:
👍

Reply:
I guess I could add a check for `rotation !== 0` in RegionAnnotations and prevent RegionCreator and ReplyPopup from showing in that case?

Reply:
Not necessary, maybe if you want you can put a comment -- we hope to fix this soon anyway